www/static/css-src/_home.scss

```diff
@@ -272,7 +272,7 @@ img#logo_top {
 }
 
 .supported_platforms {
-    background-color: darken(white, 85%);
```

Comment:
Since this is happening on a static colour `white` we should just figure out what the hex value is and use it directly or store it directly in a variable instead of needing a colour function here.